src/main/java/org/apache/ibatis/builder/xml/XMLConfigBuilder.java

```diff
@@ -372,14 +372,18 @@ private void mapperElement(XNode parent) throws Exception {
           String mapperClass = child.getStringAttribute("class");
           if (resource != null && url == null && mapperClass == null) {
             ErrorContext.instance().resource(resource);
-            InputStream inputStream = Resources.getResourceAsStream(resource);
-            XMLMapperBuilder mapperParser = new XMLMapperBuilder(inputStream, configuration, resource, configuration.getSqlFragments());
-            mapperParser.parse();
+            try (InputStream inputStream = Resources.getResourceAsStream(resource)) {
+              XMLMapperBuilder mapperParser = new XMLMapperBuilder(inputStream, configuration, resource, configuration.getSqlFragments());
+              mapperParser.parse();
+            } catch (Exception ignored) {
+            }
```

Comment:
Exceptions should not be ignored

Reply:
thank you.

Reply:
I am not a contributor just a subscriber to the mailing list, but Try with resources doesn't need a catch block, you can remove it

Reply:
yes, I also think the catch block should be removed

Reply:
thank you for telling me